pypeclub/OpenPype
# Houdini: Increment current file on workfile publish

## Brief description

This fixes #2687

## Description

Matches logic more to Maya where it also increments current file with the workfile family.
Previous plug-ins had `targets` "local" and "deadline" which I believe aren't even used in OpenPype - so have been non-functional for that time.

## Testing notes:

1. Publish with Houdini
2. Ensure scene incrementing behavior is as expected
